src/games/rcll/challenges.clp

```diff
@@ -113,7 +113,7 @@
 
 (defrule challenges-create-routes
 " Create some routes with waypoints in the field as a navigation challenge "
-	(declare (salience ?*PRIORITY_CHALLENGE_OVERRIDE*))
+	(declare (salience ?*PRIORITY_CHALLENGE-OVERRIDE*))
```

Comment:
is there a reason why we mix upper and lower case (sometimes even within one global)?